internal/util/util.go

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright Strimzi authors.
+// License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+//
+
+// Package util contains some utility functions
+package util
+
+import "time"
+
+// NowToMilliseconds returns the current time in milliseconds
+func NowToMilliseconds() int64 {
```

Comment:
I wonder if it should be called `NowInMilliseconds()` => you are not converting some passed parameter to milliseconds but instead getting the current time in milliseconds.

Reply:
Agree, it makes more sense.